lib/docx/containers/text_run.rb

```diff
@@ -45,6 +45,13 @@ def parse_text
           @text_nodes.map(&:content).join('')
         end
 
+        # Substitute text in text @text_nodes
+        def gsub!(match, substitute)

```

Comment:
The feature and your implementation looks good to me :) but its name is not intuitive. 
This is because this name makes users would expect the same interface of `String#gsub!` but not. 

`substitute` looks better, I think. how do you feel?

Reply:
Sure, substitute sounds about right to me!

Reply:
Updated and rebased the code, thanks for the review @satoryu!